NEWS.md

```diff
@@ -1,3 +1,12 @@
+# bbr 1.8.1
+
+## Bugs addressed
+
+* `inherit_param_estimates()` would occasionally fail during the `model_summary()`
+call if the previous model had not converged correctly or `$COV` has not been 
+evaluated (no `.grd` file). Added `.bbi_args` argument (similar to `nm_join()`),
+that defaults to *not* looking for `.shk` and `.grd` files. (#638)
```

Comment:
Reading this, I tripped a bit on "occasionally" (does that mean even models following under the "if ..." only fail occasionally?) and the "not converged correctly" (what are all the cases that includes?).

I think I'd find it clearer with these tweaks

```
* `inherit_param_estimates()` calls `model_summary()` underneath and that
   failed when the parent model did not have `.grd` or `.shk` files.
   `inherit_param_estimates()` now has a `.bbi_args` argument (similar to
   `nm_join()`) that defaults to *not* looking for `.shk` and `.grd`
   files. (#638)
```

... but I'm all right if you prefer to stick with what you've got.